packages/cfd/src/Containers/jurisdiction-modal/jurisdiction-modal-content-items/jurisdiction-modal-card/jurisdiction-card.tsx

```diff
@@ -0,0 +1,123 @@
+                <div className={`${card_classname}__info-container`}>
+                    <Text as='p' color={'prominent'} weight='bold' size='sm' className={`${card_classname}__h2-header`}>
+                        <Localize
+                            i18n_default_text={
+                                jurisdiction_contents[type_of_card as keyof typeof jurisdiction_contents].header
+                            }
+                        />
+                    </Text>
+                    {account_type === 'synthetic'
+                        ? jurisdiction_contents[
+                              type_of_card as keyof typeof jurisdiction_contents
+                          ].synthetic_contents.map((item, index) => (
+                              <div className={`${card_classname}__bullet-wrapper`} key={index}>
+                                  <div>
+                                      <Checkmark />
+                                  </div>
+                                  <Text as='p' size='xs' color={'prominent'}>
+                                      <Localize i18n_default_text={item} />
+                                  </Text>
+                              </div>
+                          ))
+                        : jurisdiction_contents[
+                              type_of_card as keyof typeof jurisdiction_contents
+                          ].financial_contents.map((item, index) => (
+                              <div className={`${card_classname}__bullet-wrapper`} key={index}>
+                                  <div>
+                                      <Checkmark />
+                                  </div>
+                                  <Text as='p' size='xs' color={'prominent'}>
+                                      <Localize i18n_default_text={item} />
+                                  </Text>
+                                  {/* TODO: find a better solution */}
+                                  {/Straight-through processing/.test(item) && (
+                                      <Popover
+                                          alignment='left'
+                                          className='cfd-compare-accounts-tooltip'
+                                          classNameBubble='cfd-compare-accounts-tooltip--msg'
+                                          icon='info'
+                                          disable_message_icon
+                                          is_bubble_hover_enabled
+                                          message={localize(
+                                              'Choosing this jurisdiction will give you a Financial STP account. Your trades will go directly to the market and have tighter spreads.'
+                                          )}
+                                          zIndex={9999}
+                                      />
+                                  )}
+                              </div>
+                          ))}
```

Comment:
I see no difference between account_type === 'synthetic' and not conditions apart from having a popover. If that is the case this is a better solution:


```suggestion
                    {jurisdiction_contents[type_of_card as keyof typeof jurisdiction_contents].synthetic_contents.map(
                        (item, index) => {
                            return (
                                <React.Fragment key={index}>
                                    <div className={`${card_classname}__bullet-wrapper`}>
                                        <div>
                                            <Checkmark />
                                        </div>
                                        <Text as='p' size='xs' color={'prominent'}>
                                            <Localize i18n_default_text={item} />
                                        </Text>
                                    </div>
                                    {account_type === 'synthetic' && /Straight-through processing/.test(item) && (
                                        <Popover
                                            alignment='left'
                                            className='cfd-compare-accounts-tooltip'
                                            classNameBubble='cfd-compare-accounts-tooltip--msg'
                                            icon='info'
                                            disable_message_icon
                                            is_bubble_hover_enabled
                                            message={localize(
                                                'Choosing this jurisdiction will give you a Financial STP account. Your trades will go directly to the market and have tighter spreads.'
                                            )}
                                            zIndex={9999}
                                        />
                                    )}
                                </React.Fragment>
                            );
                        }
                    )}
```